topics/researchcloud.qmd

```diff
@@ -0,0 +1,47 @@
+---
+title: SURF Research Cloud
+---
+## What is it?
+[SURF Research Cloud](https://www.surf.nl/en/services/surf-research-cloud) is a portal where you easily build a virtual research environment. You can use preconfigured workspaces and datasets or add them yourself. Institutions, research communities and suppliers can contribute to Research Cloud's functionality and catalogue by integrating their computing and data services. 
+
+SURF Research Cloud is hosted at the SURF datacenter in Amsterdam. 
+
+## What can it be used for?
+SURF Research Cloud enables you to run 1 or more servers with applications that can be accessible over the internet.
+
+- Hosting web applications, these can be connected to SRAM for secure authentication. You can easily start more servers to increase availability and preformance.
+
+- Running a desktop environment with analysis tools, use your favorite desktop tools on an environment with more performance. Unlike [SciCloud][./scicloud.qmd] GPU resources are available. You can easily connect environment to your data on Research Drive or Yoda.
+
+- Hosting a highly secure research environment where sensitive data cannot leave the VRE: [SURF SANE](https://www.surf.nl/en/themes/research-infrastructure/sane-secure-environment-for-analysing-sensitive-data).
+
+## How to request access
+All VU researchers can get access to SURF Research Cloud. To start building an environment in Research Cloud you need 2 things:
+
+**1. An SRAM Collaboration connected to Research Cloud**
+
+- First apply for a new SRAM Collaboration (CO): log in to [SRAM](https://sram.surf.nl) and click the "Request collaboration" button. 
+
+- The SRAM admins at IT for Research (ITvO) will contact you to discuss your specific needs and help you to get started.
+
+- Once the Collaboration request is approved you can connect the Research Cloud Application to the new collaboration.
+
+- Invite the colleagues that need to work in your VRE to your new CO.
+
+**2. A Wallet with credits**
+
+There are 2 ways to obtain credits:
+
+- Apply for a _NWO Small Compute Grant_ by following the links on [this page](https://www.surf.nl/en/small-compute-applications-nwo). In the request form select "SURF Research Cloud - HPC Cloud".
+
+- If for whatever reason your application for this grant is denied, you could also claim credits from the VU [RCCS](https://www.surf.nl/research-capacity-computing-service-rccs-directe-toegang-tot-onze-rekensystemen-en-expertise) contract. Please contact [IT for Research](mailto:itvo.it@vu.nl) for details on how to obtain these credits.
+
+Once the request is granted you will have access to a _wallet_ in Research Cloud and can start to build an environment.
```

Comment:
Maybe it's good to give some estimation of what these credits allow you to do? How do these credits translate to the service you get?

Reply:
I know for sure I have seen that somewhere but I can't find it in the SURF wiki.

Reply:
I can only find this in an old email I got from a colleague. I will open a ticket with SURF to ask if they have this online somewhere.

Reply:
https://servicedesk.surf.nl/jira/plugins/servlet/desk/portal/13/SD-89335

Reply:
Added